src/helpers/screenings.js

```diff
@@ -3,13 +3,52 @@ import fetch from "node-fetch";
 const API = "https://plankton-app-xhkom.ondigitalocean.app/api/screenings";
 
 export async function movieScreenings(id) {
-    const res = await fetch(`${API}?filters[movie]=${id}`);
-    const payload = await res.json();
-    return payload.data;
+  const res = await fetch(`${API}?filters[movie]=${id}`);
+  const payload = await res.json();
+  return payload.data;
 }
 
 export async function getAllScreenings() {
-    const res = await fetch(API);
-    const payload = await res.json();
-    return payload.data;
+  const res = await fetch(API);
+  const payload = await res.json();
+  return payload.data;
+}
+
+export async function screeningsStartpage() {
+  const apiRes = await fetch(API + "?populate=movie");
+  const payload = await apiRes.json();
+  const result = payload.data
+    .map((item) => ({
+      id: item.id,
+      ...item.attributes,
+      movie: {
+        id: item.attributes.movie.data.id,
+        ...item.attributes.movie.data.attributes,
+      },
+    }))
+    .filter((screenings) => {
```

Comment:
This variable (`screenings`) will get the value of a single screening, and be executed once for every screening in the `payload.data` array. So I think `screening` (singular) would be a better name.